FoundationDB/fdb-kubernetes-operator
# Run a wider build before running tests to help catch latent changes to generated files.

# Description

*Please include a summary of the change and which issue is addressed. If this change resolves an issue, please include the issue number in the description.*

This should catch latent changes that I noticed in my local environment when I ran a clean build.

## Type of change

*Please select one of the options below.*

- Bug fix (non-breaking change which fixes an issue)

# Discussion

*Are there any design details that you would like to discuss further?*

No.

# Testing

*Please describe the tests that you ran to verify your changes. Unit tests?
Manual testing?*

I'm hoping this PR will fail on the first build so we can confirm that it sees the same pending changes.

# Documentation

*Did you update relevant documentation within this repository?*

No.

# Follow-up

*Are there any follow-up issues that we should pursue in the future?*

No.

*Does this introduce new defaults that we should re-evaluate in the future?*

No.
